babel/index.babel

```diff
@@ -0,0 +1,27 @@
+// !! IMPORTANT README:
+
+// You may add additional external JS and CSS as needed to complete the project, however the current external resource MUST remain in place for the tests to work. BABEL must also be left in place. 
+
+/***********
+INSTRUCTIONS:
+  - Select the project you would 
+    like to complete from the dropdown 
+    menu.
+  - Click the "RUN TESTS" button to
+    run the tests against the blank 
+    pen.
+  - Click the "TESTS" button to see 
+    the individual test cases. 
+    (should all be failing at first)
+  - Start coding! As you fulfill each
+    test case, you will see them go   
+    from red to green.
+  - As you start to build out your 
+    project, when tests are failing, 
+    you should get helpful errors 
+    along the way!
+    ************/
+
```

Comment:
It is an easy task to get rid of unnecessary code in your project. Unnecessary code makes your project confusing and easy to be misunderstood. You can delete this file altogether as well as any instructions that are left as comments. find out more in point no. 12 in [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
- [ ] Please kindly rephrase your comment in a suggestive manner instead of an order. 🙏 

**This is a polite suggestion: "I think it would be better to use semantic tags"**

**This reads as an order: "Use semantic tags"**